daily_post.py

```diff
@@ -5,23 +5,88 @@
 import praw
 import requests
 import time
+import markovify
+from unidecode import unidecode
+import re
+import random
 
 
 from bot import *
 from bs4 import BeautifulSoup
 from jinja2 import Environment, FileSystemLoader
 
+
+STATE_SIZE = 2
+INFO_URL = 'https://github.com/trambelus/UserSim'
+SUB_URL = '/r/User_Simulator'
+SUB = 'SeattleWA'
+
+FOOTER = '\n\n-----\n\n[^^Info](%s) ^^| [^^Subreddit](%s)' % (INFO_URL, SUB_URL)
+
 reddit = praw.Reddit(client_id=CLIENT_ID,
                     client_secret=CLIENT_SECRET,
                     password=PASSWORD,
                     user_agent=USER_AGENT,
                     username=USERNAME)
 
-subreddit = reddit.subreddit('SeattleWA')
+subreddit = reddit.subreddit(SUB)
 
 now = datetime.datetime.now()
 
 
+class PText(markovify.Text):
```

Comment:
Adding classes inside of this bot script is starting to creep into one giant python file for a greater project. We may want to consider moving classes outside of the main bot executable